java/test/jmri/jmrit/display/layoutEditor/LayoutEditorAuxToolsTest.java

```diff
@@ -517,6 +519,8 @@ public static void setUpClass() throws Exception {
             JUnitUtil.initInternalTurnoutManager();
             JUnitUtil.initInternalSensorManager();
 
+            //save the old string comparator
+            stringComparator = Operator.getDefaultStringComparator();
```

Comment:
I don’t see where you are restoring this in this file.

Reply:
Thanks for the catch… (now fixed) I'm actually trying to figure things out in PR #7264. Eyeballs there would be welcome.